JASP-Desktop/widgets/boundqmllistviewterms.cpp

```diff
@@ -193,7 +193,15 @@ bool BoundQMLListViewTerms::isJsonValid(const Json::Value &optionValue)
 					}
 					const Json::Value& components = value[_tempOptionKey];
 					if (_termsModel->areTermsInteractions())
+					{
 						valid = components.type() == Json::arrayValue;
+						if (!valid)
```

Comment:
Can't you just do:
```
if((components.type == Json::stringValue)
{
	valid = true;
	Log::log() << "JASP file has a VariableList with interactions but the elements are strings instead of arrays. Probably an old JASP file." << std::endl;
}
```